### ISSUE TITLE
ReduceLROnPlateau

### ISSUE THREAD

**Issue by nbgundavarapu** [md-mohaiminul/ViS4mer#4]
ReduceLROnPlateau by default assumes a "min" metric (https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html)

`mode ([str](https://docs.python.org/3/library/stdtypes.html#str)) – One of min, max. In min mode, lr will be reduced when the quantity monitored has stopped decreasing; in max mode it will be reduced when the quantity monitored has stopped increasing. Default: ‘min’.`

In LVU tasks it used for accuracy metric, but the mode is not changed. Is this expected?
https://github.com/md-mohaiminul/ViS4mer/blob/2a2442bb0fc84b6823150c73b00e713179c59b7c/run_lvu.py#L94
https://github.com/md-mohaiminul/ViS4mer/blob/2a2442bb0fc84b6823150c73b00e713179c59b7c/run_lvu.py#L345

**Comment by md-mohaiminul** [comment]
Hi,
That's a very good observation. I guess it was not intended. You can try changing the mode of ReduceLROnPlateau to 'max'. 
Thanks
